app/editor/src/env.json

```diff
@@ -0,0 +1,4 @@
+{
+  "dev-env": "tno-dev.apps.silver.devops.gov.bc.ca",
```

Comment:
Make it an array and add the following `dev.editor.mmia.gov.bc.ca` and `test.editor.mmia.gov.bc.ca`

Reply:
Updated, thanks